File: CookMaster/Models/Ingredient.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace CookMaster.Models
{
    public class Ingredient
    {
        [PK]
        public int ID { get; set; }

        public string Aisle { get; set; }

        [NotStoredInDB]
        public double Amount { get; set; }

         public string Consitency { get; set; }

        public string Image { get; set; }
        public string Name { get; set; }
        public string Original { get; set; }
        public string OriginalName { get; set; }
        public string Unit { get; set; }

        [NotStoredInDB]
        public List<string> Meta { get; set; }

        [NotStoredInDB]
        public IngredientMeasure Measures { get; set; }

        [NotStoredInDB]
        public List<IngredientsNutrient> Nutrients { get; set; }
    }
}

File: CookMaster.WebApp/ClientApp/src/app/shared/models/ingredient.ts
import {IngredientMeasure} from './ingredient-measure';
import {IngredientsNutrient} from './ingredients-nutrient';

export interface Ingredient {
  ID: number;
  Aisle: string;
  Amount: number;
  Consitency: string;  // kept typo to match backend exactly
  Image: string;
  Name: string;
  Original: string;
  OriginalName: string;
  Unit: string;

  Meta: string[];
  Measures: IngredientMeasure;
  Nutrients: IngredientsNutrient[];
}
